content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/_cq_design_dialog/.content.xml

```diff
@@ -49,7 +49,7 @@
                                         jcr:primaryType="nt:unstructured"
                                         sling:resourceType="granite/ui/components/coral/foundation/form/multifield"
                                         class="foundation-layout-util-maximized-alt"
-                                        fieldDescription="The client-side libraries that are to be loaded in the body element.">
+                                        fieldDescription="The client-side javascript libraries that are to be loaded in the body element.">
```

Comment:
This is still  not correct, as these may also be CSS libraries (loaded in the head)

Reply:
I too think it's best to allow CSS as well in the head (with checks to avoid loading the same clientlib twice)

Reply:
Also for backwards-compatibility reasons this clientlib must be loaded for type CSS in the head (even in the future).